controllers/mapController.js

```diff
@@ -0,0 +1,24 @@
+const Map = require("../models/Map");
+const ERROR_MESSAGE = require("../constants");
+
+const getmapPoints = async (req, res, next) => {
```

Comment:
```suggestion
const getMapPoints = async (req, res, next) => {
```

카멜케이스 적용이 안된 것 같습니다!